apps/chat/src/store/import-export/importExport.epics.ts

```diff
@@ -299,12 +299,15 @@ const importConversationsEpic: AppEpic = (action$) =>
                 entities: conversationsListing,
               });
             });
+          const emptyFolders = folders.filter((folder) =>
```

Comment:
Why it's named `empty`?

Reply:
This is name for the folders without conversations inside.